src/client/base.js

```diff
@@ -286,6 +287,7 @@ spf.MultipartResponse;
  *       is set to "POST".
  *
  * @typedef {{
+ *   headers: (Object.<string, string>|undefined),
```

Comment:
You can probably make this just Object.<string> for consistency with how you cast the data below.